src/plaster/loaders.py

```diff
@@ -61,64 +69,79 @@ def setup_logging(config_uri, defaults=None):
     module. However, it may also be used to configure any other logging
     subsystems that serve a similar purpose.
 
-    ``config_uri`` may be anything that can be parsed by
-    :func:`plaster.parse_uri`.
+    :param config_uri: Anything that can be parsed by
+        :func:`plaster.parse_uri`.
+
+    :param defaults: A ``dict`` of default values used to populate the
+        settings and support variable interpolation. Any values in ``defaults``
+        may be overridden by the loader prior to returning the final
+        configuration dictionary.
 
     """
     loader = get_loader(config_uri)
     return loader.setup_logging(defaults)
 
 
-def get_loader(config_uri, protocol=None):
+def get_loader(config_uri, protocols=None):
     """
     Find a :class:`plaster.ILoader` object capable of handling ``config_uri``.
 
-    ``config_uri`` may be anything that can be parsed by
-    :func:`plaster.parse_uri`.
+    :param config_uri: Anything that can be parsed by
+        :func:`plaster.parse_uri`.
+
+    :param protocols: Zero or more :term:`loader protocol` identifiers that
+        the loader must implement to match the desired ``config_uri``.
 
-    ``protocol`` may be a :term:`loader protocol` that the loader must
-    satisfy to match the desired ``config_uri``.
+    :returns: A :class:`plaster.ILoader` object.
+    :raises plaster.LoaderNotFound: If no loader could be found.
+    :raises plaster.MultipleLoadersFound: If multiple loaders match the
+        requested criteria. If this happens, you can disambiguate the lookup
+        by appending the package name to the scheme for the loader you wish
+        to use. For example if ``ini`` is ambiguous then specify
+        ``ini+myapp`` to use the ini loader from the ``myapp`` package.
 
     """
     config_uri = parse_uri(config_uri)
     requested_scheme = config_uri.scheme
 
-    matched_loaders = find_loaders(requested_scheme, protocol=protocol)
+    matched_loaders = find_loaders(requested_scheme, protocols=protocols)
 
     if len(matched_loaders) < 1:
```

Comment:
Ah. Just made me run the line through my head twice while looking at it :P wondering if there was something I was missing.